python/tvm/driver/tvmc/compiler.py

```diff
@@ -283,7 +289,7 @@ def save_module(module_path, graph, lib, params, cross=None):
         lib.export_library(path_lib)
     else:
         logger.debug("exporting library to %s , using cross compiler %s", path_lib, cross)
-        lib.export_library(path_lib, cc.cross_compiler(cross))
+        lib.export_library(path_lib, cc.cross_compiler(cross, options=cross_options.split(' ')))
```

Comment:
Maybe it makes sense to add the command line options to the debug message on the line above?